tests/test_bot_commands.py

```diff
@@ -0,0 +1,63 @@
+import os
+import socket
+
+import discord
+import discord.ext.commands as commands
+import discord.ext.test as dpytest
+import pytest
+import pytest_asyncio
+
+from scutoid import Scutoid
+from util import read_yaml
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+REAL_NAMES = read_yaml(os.path.join(ROOT_DIR, "../real_names.yaml"))
+
+
+@pytest_asyncio.fixture
+async def bot():
+    # Setup
+    intents = discord.Intents.default()
+    intents.members = True
+    intents.message_content = True
+    nicknamer = commands.Bot(command_prefix="!", intents=intents)
+
+    # noinspection PyProtectedMember
+    await nicknamer._async_setup_hook()
+    await nicknamer.add_cog(Scutoid(nicknamer, REAL_NAMES))
+
+    dpytest.configure(nicknamer)
+
+    yield nicknamer
+
+    # Teardown
+    await dpytest.empty_queue()  # empty the global message queue as test teardown
+
+
+@pytest.mark.asyncio
+async def test_nick(bot):
+    # GIVEN
+    self_member = bot.guilds[0].me
```

Comment:
what user is this? maybe comment

Reply:
It doesn't really matter.  I'm actually just getting some random member here. The test harness default initializes the guild with some members and context.